modules/price/src/Plugin/Field/FieldFormatter/PriceDefaultFormatter.php

```diff
@@ -142,6 +142,9 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
             'languages:' . LanguageInterface::TYPE_INTERFACE,
             'country',
           ],
+          'tags' => [
+            'config:' . $items[$delta]->getFieldDefinition()->getTargetEntityTypeId() . '.' . $items[$delta]->getFieldDefinition()->getTargetBundle() . '.' . $this->viewMode,
```

Comment:
$this->fieldDefinition should be the same as $items[$delta]->getFieldDefinition(), right? That one is shorter.
Looks like this tag doesn't change between deltas, which means we can calculate it above the loop (define a $cache_tag above $elements = [];)